client/web/src/cody/onboarding/CodyOnboarding.tsx

```diff
@@ -147,16 +157,16 @@ function PurposeStep({ onNext }: { onNext: () => void }): JSX.Element {
                     tabIndex={0}
                     className="flex-1 d-flex flex-column justify-content-center cursor-pointer align-items-center py-3 px-2"
                     onKeyDown={() => {
-                        eventLogger.log(EventName.CODY_ONBOARDING_PURPOSE_PERSONAL)
+                        eventLogger.log(EventName.CODY_ONBOARDING_PURPOSE_PERSONAL, { useCase: 'personal' })
```

Comment:
Should this be this?
```suggestion
                        eventLogger.log(EventName.CODY_ONBOARDING_PURPOSE_SELECTED, { useCase: 'personal' })
```